packages/babel-preset-env/src/normalize-options.js

```diff
@@ -179,6 +179,15 @@ export default function normalizeOptions(opts: Options) {
 
   checkDuplicateIncludeExcludes(include, exclude);
 
+  let polyfills = validatePolyfillsOption(opts.polyfills);
+  let useBuiltIns;
+  if (typeof opts.useBuiltIns === "string") {
+    console.warn("'useBuiltIns' has been renamed to 'polyfills'.");
```

Comment:
I don't think a warning is workable if we're landing this during the lifetime of 7.x. If it were still before the 7.x release maybe, but not during.